src/accelerate/utils/dataclasses.py

```diff
@@ -972,6 +972,16 @@ class DeepSpeedPlugin:
             " `MixtralSparseMoeBlock`, `Qwen2MoeSparseMoeBlock`, `JetMoEAttention,JetMoEBlock` ..."
         },
     )
+    enable_msamp: bool = field(
+        default=None,
+        metadata={"help": "Flag to indicate whether to enable MS-AMP backend for FP8 training."},
+    )
```

Comment:
In the next PR I'll be propagating this to the `FP8RecipeKwargs` since we can have both MS-AMP and TransformersEngine happening at once